### DISCUSSION TITLE
[New Feature] Customisable options for the Drop Pod Superweapon.

### DISCUSSION THREAD

**Discussion by CCHyper** [Vinifera-Developers/Vinifera#1487] [Feature Requests]
### Description:
Add customisable options for the Drop Pod Superweapon.

### Possible Implementation:
New INI keys for the SuperWeaponType definition;

These would control which objects can be dropped by the special;
`DropPodFirstObject=<TechnoType>` 
`DropPodSecondObject=<TechnoType>`

`DropPodFirstMinMax=<min, max>` Minimum and Maximum amount of the first object type.
`DropPodSecondMinMax=<min, max>` Minimum and Maximum amount of the second object type.

The following are global settings in `RULES.INI` that could also be defined on the SuperWeaponType itself;
`DropPodHeight=<int>`
`DropPodSpeed=<int>`
`DropPodAngle=<float>`
`DropPodWeapon=<WeaponType>`

**Comment by tomsons26** [comment]
List instead of first second perhaps?
